docs/htdocs/info/docs/tools/vep/vep_formats.html

```diff
@@ -294,6 +294,20 @@ <h2 id="hgvs"> HGVS identifiers</h2>
     <pre class="code sh_sh">LRG_1t1:c.841G>T
 LRG_1:g.10006G>T</pre>
 
+    <h4>Ambiguous gene-based descriptions</h4>
+    <p>It is possible to use ambiguous descriptions listing only gene symbol or UniProt accession and protein change (e.g. PHF21B:p.Tyr124Cys,  P01019:p.Ala268Val), as seen in the literature, though this is not recommended as it can produce multiple different variants at genomic level. The transcripts for a gene are considered in the following order:
+      <ol>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Select transcript status</a></li>
+        <li><a href="http://www.ensembl.org/info/genome/genebuild/mane.html">MANE Plus Clinical transcript status</a></li>
```

Comment:
Is this information only relevant to the www site? If so, please change the protocol to https - if not, make it a relative link.

Reply:
I don't see what's changed here - it should either be "https://www.ensembl.org/info/genome/genebuild/mane.html" or "/info/genome/genebuild/mane.html"